### DISCUSSION TITLE
More info about EXTENSIONS_AUTO_RELOAD ?

### DISCUSSION THREAD

**Discussion by Silbad** [directus/directus#10921] [Archive]
Hello,
Can we have more info about this new config => EXTENSIONS_AUTO_RELOAD ?

**Comment by nickrum** [comment]
Sure! It's an environment variable that, when set to `true`, will instruct Directus to automatically reload any extension when the extension source has changed or an extension has been added or removed. Let's add this info to the docs in #10922.

**Reply by Eliophot** [reply]
So basically, with this variable, no need to restart the server for the extension to be taken into effect?

**Reply by rijkvanzanten** [reply]
Exactly!

**Reply by Eliophot** [reply]
PERFECT !!!!
